### ISSUE TITLE
Feature Request: Change mesh size from original version

### ISSUE THREAD

**Issue by GroMMo** [milkdrop2077/MilkDrop3#17]
Could we adjust the mesh size please?  I loved crippling computers by increasing the mesh size and seeing how far you could push a graphics card using Milkdrop.

**Comment by milkdrop2077** [comment]
Yes I will allow it on the next update, I'm working enabling all the option back + new ones in the ini file

**Comment by GroMMo** [comment]
Love your work, thank you for doing this

**Comment by milkdrop2077** [comment]
You can change the mesh size from MilkDrop 3.21 with the MilkDropPanel!

**Comment by GroMMo** [comment]
Love it, I grabbed the new copy 6 hrs after upload.  Love what you have done with the interface and having the ini file rocks.  Thank you for letting me cripple my machine YAY!
